src/main/java/ManageInputs.java

```diff
@@ -0,0 +1,209 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class ManageInputs {
+    protected static String from;
+    protected static String to;
+    public static String description;
+    protected static String by;
+
+    private static void writeToFile(String filePath, Task textToAdd) throws IOException {
+        FileWriter fw = new FileWriter(filePath, true);
+        fw.write(textToAdd + "\n");
+        fw.close();
+    }
+
+    private static int fillFileContents(ArrayList<Task> tasks, String filePath, int index) throws IOException, UnexpectedCommandException {//updates index
+        File f = new File(filePath); // create a File for the given file path
+        Scanner s = new Scanner(f); // create a Scanner using the File as the source
+
+        while (s.hasNext()) {
+            String sLine = s.nextLine();
+            if (sLine.contains("[E]")) {
+                dealWithEvent(tasks, index, sLine);
+            } else if (sLine.contains("[D]")) {
+                dealWithDeadline(tasks, index, sLine);
+            } else if (sLine.contains("[T]")) {
+                dealWithTodo(tasks, index, sLine);
+            }
+            index++;
```

Comment:
This is a good application of SLAP! But perhaps the variable `sLine` could be renamed to make it clearer what this means?